src/barnetilsyn/steg/1-omdeg/OmDeg.tsx

```diff
@@ -87,9 +99,10 @@ const OmDeg: FC = () => {
     søknad.medlemskap
   );
 
-  const søkerBorPåRegistrertAdresse =
-    søknad.søkerBorPåRegistrertAdresse &&
-    søknad.søkerBorPåRegistrertAdresse.verdi === true;
+  const søkerBorPåRegistrertAdresseEllerHarMeldtAdresseendring =
+    søknad.person.søker.erStrengtFortrolig ||
+    søknad.søkerBorPåRegistrertAdresse?.verdi === true ||
```

Comment:
```suggestion
    søknad.søkerBorPåRegistrertAdresse?.verdi ||
    søknad.adresseopplysninger?.harMeldtAdresseendring?.verdi;
```

Tror man kan sløyfe `=== true` her?

Reply:
Då blir det vel at den returnerer `boolean | undefined`, tenker det kan være fint at den kun returnerer en boolean